net/uradvd/files/etc/config/uradvd

```diff
@@ -0,0 +1,13 @@
+
+# simple IPv6 SLAAC advertisement
+config interface
+	# Interface to advertise on
+	option ifname 'br-lan'
+	# default lifetime in seconds (0: forever)
```

Comment:
The comment is wrong. `0` means a lifetime of 0, meaning that no default route is announced.